src/screens/FeedScreen/Content.js

```diff
@@ -377,12 +391,11 @@ export const styles = StyleSheet.create({
     marginLeft: 8,
     marginRight: 8
   },
-  contentFeed: (hasComment) => ({
-    flex: !hasComment ? undefined : 1,
+  contentFeed: {
+    flex: 1,
     marginTop: 0,
-    height: !hasComment ? dimen.normalizeDimen(355) : '100%',
     width: '100%'
-  }),
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for the feed content no longer depends on whether there are comments. This simplifies the style, but make sure this doesn't cause any visual issues or overlaps between components.
```diff
-  contentFeed: (hasComment) => ({
-    flex: !hasComment ? undefined : 1,
-    height: !hasComment ? dimen.normalizeDimen(355) : '100%',
-    width: '100%'
-  }),
+  contentFeed: {
+    flex: 1,
+    marginTop: 0,
+    width: '100%'
+  },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->